source/appModules/foobar2000.py

```diff
@@ -14,31 +14,38 @@
 # A named tuple for holding the elapsed and total playing times from Foobar2000's status bar
 statusBarTimes = collections.namedtuple('StatusBarTimes', ['elapsed', 'total'])
 
-def getParsingFormat(interval):
-	"""Attempts to find a suitable parsing format string for a HH:MM:SS, MM:SS or SS -style time interval."""
-	timeParts = len(interval.split(":"))
+
+def parseIntervalToTimestamp(interval):

```

Comment:
please add type hints

```suggestion
def parseIntervalToTimestamp(interval: str): -> Optional[int]
```